apptentive/src/com/apptentive/android/sdk/module/engagement/interaction/model/TextModalInteraction.java

```diff
@@ -24,8 +24,8 @@
 	public static final String EVENT_NAME_DISMISS = "dismiss";
 	public static final String EVENT_NAME_INTERACTION = "interaction";
 
-	public static final String EVENT_KEY_LABEL = "label";
```

Comment:
I switched to using the already present key with the same value from the Target object.